docker/python3/Dockerfile

```diff
@@ -1,6 +1,8 @@
 # Last modified: 2024-11-14T00:16:47.268736+00:00
 FROM python:3.11.10-alpine3.20
 
+
+
```

Comment:
Why 2 extra lines?

Reply:
that's how i do my testing, by changing the image i'm causing it to be built, will be removed befre the merge